### PR TITLE
feat: support bracket with in operator

### PR DESCRIPTION
**Pull request by closetool**
Signed-off-by: closetool <c299999999@qq.com>

Fix: #863

### CODE REVIEW

**Comment by hsluoyz** [comment]
@closetool AT them to review

**Comment by closetool** [comment]
@tangyang9464 @Zxilly please review

**Review by Zxilly — APPROVED** [review]
lgtm

**Comment by Zxilly** [comment]
@closetool You can also add a unit test with `[]` grammar, to prevent potential break change in the future.

**Review by tangyang9464 — APPROVED** [review]
LGTM

**Comment by hsluoyz** [comment]
> @closetool You can also add a unit test with `[]` grammar, to prevent potential breaking change in the future.

good point, @closetool

**Comment by closetool** [comment]
@hsluoyz @Zxilly Added

File: examples/rbac_model_matcher_using_in_op_bracket.conf [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+p = sub, obj, act
+
+[role_definition]
+g = _, _
+
+[policy_effect]
+e = some(where (p.eft == allow))
+
+[matchers]
+m = g(r.sub, p.sub) && r.obj == p.obj && r.act == p.act || r.obj in ('data2', 'data3')
```
Comment (Zxilly) [inline examples/rbac_model_matcher_using_in_op_bracket.conf:14]: should be square bracket

**Review by Zxilly — CHANGES_REQUESTED** [review]

File: main/go.mod [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module github.com/closetool/playground
+
+go 1.16
+
+require github.com/casbin/casbin/v2 v2.35.0
+
+replace github.com/casbin/casbin/v2 => ../
```
Comment (Zxilly) [inline main/go.mod:7]: Plz remove these files

**Review by Zxilly — CHANGES_REQUESTED** [review]

**Pull request merged**